### PR TITLE
Added Azure Pipelines support for CI builds

### PR DESCRIPTION
**Pull request by CreepPork**
**When merged this pull request will:**
- Add a makefile for building PBOs
- Add Azure Pipelines support
- Add `getArmake.ps1` PowerShell script for Windows users to download armake
~~Play around and test with Travis~~
- [x] Add notifications

### CODE REVIEW

**Comment by jonpas** [comment]
armake (1) is no longer supported!

Use HEMTT and you don't need a long Makefile or a PowerShell download script.

**Comment by CreepPork** [comment]
But armake is more stable than armake2 and doesn't require any more changes to the code base than proposed here.

**Comment by dedmen** [comment]
Then you might aswell look at armake++ which is faster and has even more bugs fixed.
https://github.com/dedmen/armake/releases
not sure how up-to-date the release files are compared tothe code.

**Comment by CreepPork** [comment]
I'm not sure I should migrate over to armake++, it's plenty fast enough, the build time is less than 1 minute and for now, no bugs have appeared, so it's pretty stable.

File: addons/main/script_version.hpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
 #define MAJOR 1
 #define MINOR 1
 #define PATCHLVL 0
-#define BUILD 4
+#define BUILD 4
```
Comment (mharis001) [inline addons/main/script_version.hpp]: Newline?

Comment (CreepPork) [reply addons/main/script_version.hpp]: The new line is stripped by the makefile, I’ll see if I can append `\n`.

Comment (jonpas) [reply addons/main/script_version.hpp]: You are appending a newline at the others, why not the last one.

**Pull request merged**